testdata/test

```diff
@@ -1,11 +1,7 @@
 show-widgets
 showWidgets
-betaUi
```

Comment:
this alias of `beta-ui` is what was removed, so we try to let people know

Reply:
I'm leaning toward option 2 (without the check mark) out of the two options. The skull & check mark are a little confusing. 

I like Kerrie's idea of adding a new column for all references that have been removed.

2 options:
- Column can be True or False. Are all references removed or not. 
- Column can be a count of the references found/remaining. For extinct flags, the colmn entry would be zero.